docs/concepts/event-driven-architecture.md

```diff
@@ -35,17 +35,18 @@ To implement event-driven architecture in your Amplication project, you'll need
 
 1. Create a message broker
 2. Define topics
-3. Connect services to the message broker
-4. Install the Kafka plugin
-5. Implement event production and consumption logic
+3. Connect services to the message broker and relevant topics
```

Comment:
@dericksozo - You removed the Kafka installation, but still as part of the flow, the user needs to install a message broker plugin- whether it's Kafka, Redis or any other relevant plugin. We need to mention it as part of the flow